src/utils/helpers.ts

```diff
@@ -20,3 +20,7 @@ export const getNumericLevel = (level: Level) => {
       return -1; // Invalid level
   }
 };
+
+export const ErrorDiff = (message: unknown) => {
+  return "```diff\n- " + message + "\n```";
```

Comment:
This is a neat implementation thanks for this. Would you happen to have an example comment of this error rendering?

Reply:
I just replaced the tableComment with the ErrorDiff to demo it

https://github.com/Keyrxng/UbiquityBotTesting/issues/43#issuecomment-1712851989